src/commands/InitCommand.ts

```diff
@@ -202,7 +225,7 @@ export class InitCommand implements yargs.CommandModule {
                 sourceMap: true
             }
         }
-        , undefined, 3);
+            , undefined, 3);
```

Comment:
Can you undo this change?

Reply:
Sure, but it is not done by me. VScode auto formatter make this change on saving the file. This change will not affect the code.